doc/50_IDE.md

```diff
@@ -15,11 +15,11 @@ emphasizes diversity.
 
 ## Overview
 
-When first opened, you will see a welcome screen that briefly gives you the most important pointers
-to get started. After closing it you will see the workbench shell. Depending on the
+When first open Theia, you will see a welcome screen that quickly gets you started.
```

Comment:
When _you_ first open

Reply:
fixed that independently in the follow-up